_includes/pbc/all/install-features/202410.0/install-the-payments-feature.md

```diff
@@ -24,19 +27,254 @@ To start the feature integration, overview and install the necessary features:
+The key is the payment method key, and the value is the shipment method key.
+
+**src/Pyz/Zed/PaymentAppShipment/PaymentAppShipmentConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\PaymentAppShipment;
+
+use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Zed\PaymentAppShipment\PaymentAppShipmentConfig as SprykerPaymentAppShipmentConfig;
+
+class PaymentAppShipmentConfig extends SprykerPaymentAppShipmentConfig
+{
+    /**
+     * @var array<string, string>
+     */
+    protected const EXPRESS_CHECKOUT_SHIPMENT_METHODS_INDEXED_BY_PAYMENT_METHOD = [
+        'payone-paypal-express' => 'spryker_dummy_shipment-standard',
+    ];
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+1. Add several products to cart and proceed to the express checkout flow.
+2. Use the express checkout widget to place an order shown on the cart page.
+3. Confirm the order, which opens the order confirmation page.
+  Make sure the shipment methods you've configured to be used by default was applied to the order.
+
+{% endinfo_block %}
+
+6. Enable the express checkout shipment for product bundles.
+
+**src/Pyz/Zed/PaymentAppShipment/PaymentAppShipmentConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\PaymentAppShipment;
+
+use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Zed\PaymentAppShipment\PaymentAppShipmentConfig as SprykerPaymentAppShipmentConfig;
+
+class PaymentAppShipmentConfig extends SprykerPaymentAppShipmentConfig
+{
+    /**
+     * @var list<string>
+     */
+    protected const SHIPMENT_ITEM_COLLECTION_FIELD_NAMES = [
+        QuoteTransfer::BUNDLE_ITEMS,
+    ];
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Add a product bundle to cart and place the order using the express checkout flow.
+  Make sure the order is placed successfully.
+
+{% endinfo_block %}
+
+7. When customers start a regular checkout, the quote is cleaned up to let them to restart the checkout process from the beginning. Enable cart cleanup for the regular checkout flow.
+
+
+**src/Pyz/Zed/PaymentAppShipment/PaymentAppShipmentConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\PaymentAppShipment;
+
+use Generated\Shared\Transfer\QuoteTransfer;
+use SprykerShop\Yves\CheckoutPage\Plugin\Router\CheckoutPageRouteProviderPlugin;
+use Spryker\Zed\PaymentAppShipment\PaymentAppShipmentConfig as SprykerPaymentAppShipmentConfig;
+
+class PaymentAppShipmentConfig extends SprykerPaymentAppShipmentConfig
+{
+    /**
+     * @return string
+     */
+    public function getExpressCheckoutStartPageRouteName(): string
+    {
+        return CheckoutPageRouteProviderPlugin::ROUTE_NAME_CHECKOUT_INDEX;
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+
+1. Add several products to cart and proceed to the express checkout flow.
+2. On the summary page, click the **Back to cart** button.
+3. To proceed to the regular checkout, click **Checkout**.
+  This opens the address step of the regular checkout.
```

Comment:
What do I need to verify here?

Reply:
I think it is fine as the final step of this is that the address page is opened which is bringing the customer into the "regular" checkout process.